vscode/egglog-language-server/tree-sitter-egglog/src/tree_sitter/parser.h

```diff
@@ -0,0 +1,224 @@
+#ifndef TREE_SITTER_PARSER_H_
```

Comment:
Quick question before the egglog team takes a look at this: do we have to commit these header and c files? Can they be re-generated each time?

Reply:
I think it's possible to generate those files each time perhaps using custom `build.rs`. However, this makes it harder for people just need the parser only.
So, I think it's good to commit generated files and it seems a standard way in the tree-sitter community e.g. https://github.com/tree-sitter/tree-sitter-c